### DISCUSSION TITLE
Disable user login

### DISCUSSION THREAD

**Discussion by philskents** [jordan-dalby/ByteStash#96] [Q&A]
Until SSO is implemented is there a way to bypass user login entirely so that I can just use a transparent authentication proxy in front of ByteStash?

**Comment by jordan-dalby** [comment]
@philskents 
On version 1.4.0 you can leave out the AUTH_USERNAME and AUTH_PASSWORD to disable authentication.

On the latest main, you cannot currently turn authentication off. I can make it so that if you don't set TOKEN_EXPIRY, the token will never expire, does that work for you for now?

**Reply by jordan-dalby** [reply]
I've been looking at how SSO could be implemented and actually it should be fairly straightforward. I'll move it to 1.5.0.

**Reply by jordan-dalby** [reply]
SSO is done, at least the initial version. It supports OIDC routes, tested with Authentik.
